pygameMenu/menu.py

```diff
@@ -96,6 +98,7 @@ def __init__(self,
                  height,
                  width,
                  title,
+                 auto_center_v=False,
```

Comment:
call it `center_content` to be consistent with associated method

Reply:
epic!